client/ayon_traypublisher/plugins/publish/collect_clip_instances.py

```diff
@@ -33,4 +49,37 @@ def process(self, instance):
                 instance.context.data["editorialSourcePath"])
             instance.data["families"].append("trimming")
 
+        if repres := instance.data.pop("prep_representations", None):
+            representations = []
+            for repre in repres:
+                tags = repre.get("tags", [])
+                custom_tags = repre.get("custom_tags", [])
+                content_type = repre["content_type"]
+
+                if content_type == "thumbnail":
+                    tags.append("thumbnail")
+
+                # single file type should be a string
+                new_repre_files = files = repre["files"]
+                if content_type != "image_sequence":
+                    new_repre_files = files[0]
```

Comment:
Unnecessary `files` variable, and the files should not be list even if it is single sequence file.
```suggestion
                new_repre_files = repre["files"]
                if len(new_repre_files) == 1:
                    new_repre_files = new_repre_files[0]
```